src/service-card.js

```diff
@@ -0,0 +1,71 @@
+import { LitElement, html, css } from 'lit-element/lit-element.js';
+
+class ServiceCard extends LitElement {
+  static get tag() {
+    return 'service-card';
+  }
+
+  static get properties() {
+    return {
+      image: { type: String },
+      title: { type: String },
+      info: { type: String },
+      alt: { type: String },
+      uniformPadding: { type: String, attribute: 'uniform-padding' },
+    };
+  }
+
+  constructor() {
+    super();
+    this.title = 'A shorter title';
+    this.info = 'Here is the information about the thing';
+    this.uniformPadding = '2';
+  }
+
+  static get styles() {
+    return css`
+      :host {
```

Comment:
`:host` means "this tag itself" so then it's good to set a default display as for how it should be rendered / organized. inline-flex is good baseline so that the cards all sit next to each other (and ppl can always override as needed)